xbmc/Application.cpp

```diff
@@ -1148,7 +1148,7 @@ bool CApplication::Initialize()
   else //No GUI Created
   {
     CJSONRPC::Initialize();
-    CServiceBroker::GetServiceAddons().StartBeforeLogin();
+    CServiceBroker::GetServiceAddons().Start();
```

Comment:
This would be incorrect too as it runs regardless of whether login is used (if window manage fails). Right?